tests/system/test_cluster/test_agent_files_deletion/test_agent_files_deletion.py

```diff
@@ -56,21 +75,20 @@ def test_agent_files_deletion(register_agent):
             assert result, f'This file should exist in {host} but could not be found: ' \
                            f'{file["path"].format(id=agent_id, name=agent_name)}'
 
-    # Check that agent information exists in global.db
+    # Check that agent information is in the wdb socket
+    query = f'global sql select * from agent where id={agent_id}'
```

Comment:
The same as above, make sure to also test whether the content of the `belong` table was updated.